### ISSUE TITLE
0_preliminary stage, an error occured that is ase.calculators.calculator.BadConfiguration: No configuration of espresso

### ISSUE THREAD

**Issue by siinghim** [luigibonati/DEAL#1]
Before running the case in the notebook, I install the requirements, including `flare`, `mace `, `qe` and `ammps`.

In the `0_preliminary` stage, I generate the `input-otf.yaml` without  changing the code in `flare-md.ipynb`. Then performing on-the-fly training and MD by `flare-otf input-otf.yaml`. However, an error occured which is `ase.calculators.calculator.BadConfiguration: No configuration of espresso`.   

Is that meaning any wrong in the `input-otf.yaml` or the config file?

**Comment by siinghim** [comment]
It seems the problem of  the version of ASE. I change version from 3.23 to 3.22.

**Comment by luigibonati** [comment]
Yes there have been changes in `ASE==3.23.*` which might not be compatible with the other software. I will add it in the requirements in the README, thank you! 

If you still have issues (with quantum espresso) in `ASE=3.22.*` you can use this modified version where i fixed a couple of things
https://github.com/luigibonati/ase , branch `3.22.1-fix`

**Comment by siinghim** [comment]
After changing the `ASE` version from `3.32` to `3.22`, this error will not occur. 

In `ASE==3.23.*` the calculator needs the parameters `profile`, which should be specified by `ase.calculators.espresso.EspressoProfile`

Thank you!
